lib/sidekiq/config.rb

```diff
@@ -30,15 +30,19 @@ class Config
       },
       dead_max_jobs: 10_000,
       dead_timeout_in_seconds: 180 * 24 * 60 * 60, # 6 months
-      reloader: proc { |&block| block.call }
+      reloader: proc { |&block| block.call },
+      backtrace_cleaner: ->(backtrace) { backtrace }
     }
 
     ERROR_HANDLER = ->(ex, ctx) {
       cfg = ctx[:_config] || Sidekiq.default_configuration
       l = cfg.logger
       l.warn(Sidekiq.dump_json(ctx)) unless ctx.empty?
       l.warn("#{ex.class.name}: #{ex.message}")
-      l.warn(ex.backtrace.join("\n")) unless ex.backtrace.nil?
+      unless ex.backtrace.nil?
+        backtrace = cfg[:backtrace_cleaner].call(ex.backtrace)
+        l.warn(backtrace.join("\n"))
+      end
```

Comment:
I was considering cleaning the backtrace when handling exceptions triggers an error too: https://github.com/fatkodima/sidekiq/blob/4a255eb6ff8299e0be1f3d13d46b9513e87aa6e8/lib/sidekiq/config.rb#L267-L270, would that make sense to do, or is it better to just let that log everything since it's likely an implementation issue there?